src/EditorFeatures/VisualBasicTest/Squiggles/ErrorSquiggleProducerTests.vb

```diff
@@ -78,7 +78,7 @@ End Class")
                 Dim expectedToolTip = New ContainerElement(
                     ContainerElementStyle.Wrapped,
                     New ClassifiedTextElement(
-                        New ClassifiedTextRun(ClassificationTypeNames.Text, "BC30002"),
+                        New ClassifiedTextRun(ClassificationTypeNames.Text, "BC30002", QuickInfoHyperLink.TestAccessor.CreateNavigationAction(New Uri("https://msdn.microsoft.com/query/roslyn.query?appId=roslyn&k=k(BC30002)", UriKind.Absolute)), "https://msdn.microsoft.com/query/roslyn.query?appId=roslyn&k=k(BC30002)"),
```

Comment:
> 

src\EditorFeatures\VisualBasicTest\Squiggles\ErrorSquiggleProducerTests.vb(81,88): error BC30451: 'QuickInfoHyperLink' is not declared. It may be inaccessible due to its protection level.

I can't see why it's not accessible.
The ctor is internal, but the project has InternalsVisibleTo that refers to this test project.

https://github.com/dotnet/roslyn/blob/cf55f3a58e47298426fa971d3bd9d8857c746c65/src/EditorFeatures/Core/Microsoft.CodeAnalysis.EditorFeatures.csproj#L88

Reply:
Looks like you're missing a using for `Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.QuickInfo`, which the C# version has.

Reply:
Oh. I assumed the IDE will suggest if that is missing. Probably the solution wasnt fully loaded.